sql/gen-sql-markdown.py

```diff
@@ -218,9 +236,68 @@ def generate_sql_markdown(jvm, path):
             mdfile.write("<br/>\n\n")
 
 
+def generate_sql_configs_table(jvm, path):
+    """
+    Generates an HTML table at `path` that lists all public SQL
+    configuration options.
```

Comment:
Can you add some words to explain or the outline of the generated HTML table here?